.github/workflows/build-revive-debian.yml

```diff
@@ -0,0 +1,30 @@
+name: Build revive-debian
+on:
+  workflow_dispatch:
+
+env:
+  REVIVE_DEBIAN_PACKAGE: revive-debian-x86
+  DEBIAN_CONTAINER: revive-builder-debian-x86
+  DEBIAN_CONTAINER_BUILDER: build-debian-builder.sh
+  DEBIAN_CONTAINER_RUNNER: run-debian-builder.sh
+  REVIVE_DEBIAN_INSTALL: ${{ github.workspace }}/target/release
+  REVIVE_DEBIAN_BINARY: resolc
+jobs:
+  build-revive-debian-x86:
+    name: debian-container-x86
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: build-container
+        run: |
+           (cd utils && ./${{ env.DEBIAN_CONTAINER_BUILDER}} . )
```

Comment:
why did you decide to use your own docker. With this [ci-unified] (https://github.com/paritytech/scripts/blob/master/dockerfiles/ci-unified/README.md)
you have rust already installed and you can control the versions using docker tags.
The only issue which I see is that they are using debian 11 and remix backend is based on debian 12.

Reply:
This indicates that the remix backend does also not use `ci-unifed`? But it's a good point, the debian version in the builder should probably match the version where the binary is run.

Note that if we settle on ci-unifed the supported rust version is 1.77 which is quite old. It'll probably able to build revive but I didn't try that.

Reply:
What is the issue with installing rust in a Debian container?

Reply:
I don't see any issue; I just prefer to avoid maintaining additional tools/images if I have something ready to use.